landoui/dockerflow.py

```diff
@@ -16,8 +19,15 @@ def heartbeat():
     and return a 200 iff those services and the app itself are
     performing normally. Return a 5XX if something goes wrong.
     """
-    # TODO check backing services and update tests
-    return '', 200
+    try:
+        response = requests.get(
+            current_app.config['LANDO_API_URL'] + '/__lbheartbeat__'
```

Comment:
I think it would make this PR a lot simpiler and lessen the amount of changes if we just use os.getenv and remove all the references to app.config['LANDO_API_URL'].

app.config is only needed for configuring the internals of the flask app, not for storing our custom data like the lando api url.